.travis.yml

```diff
@@ -11,7 +17,8 @@ install:
   - bower install
   - ./bin/select-ember-version.js
```

Comment:
ok. i'll remove
```
env:
   matrix:
```
from travis.yml because it's purpose is to select the ember versions